src/codebuild_environment.py

```diff
@@ -0,0 +1,68 @@
+import os
+import re
+
+
+class GitHubRepositoryURLNotFound(Exception):
+    pass
+
+
+def get_codebuild_build_arn():
+    """
+    Get env variable CODEBUILD_BUILD_ARN
+
+    :return: value or empty string if not set
+    """
+    return os.getenv("CODEBUILD_BUILD_ARN", "")
+
+
+def get_github_repo_url():
+    """
+    Get env variable CODEBUILD_SOURCE_REPO_URL
+
+    :return: value or empty string if not set
+    """
+    # Example: "https://github.com/aws/deep-learning-containers.git"
+    return os.getenv("CODEBUILD_SOURCE_REPO_URL")
```

Comment:
why empty string default for CODEBUILD_BUILD_ARN but not for this one?

Reply:
I copy pasted the getenv CODEBUILD_BUILD_ARN from
https://github.com/aws/deep-learning-containers/blob/16afa88d995b6f6afcaca3c80cba3d251e72eace/src/utils.py#L40

However, I copy-pasted the getenv for CODEBUILD_SOURCE_REPO_URL from
https://github.com/aws/deep-learning-containers/blob/16afa88d995b6f6afcaca3c80cba3d251e72eace/src/send_status.py#L77
and
https://github.com/aws/deep-learning-containers/blob/16afa88d995b6f6afcaca3c80cba3d251e72eace/src/utils.py#L130